Extension/src/LanguageServer/copilotContextTelemetry.ts

```diff
@@ -0,0 +1,72 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import { randomUUID } from 'crypto';
+import * as telemetry from '../telemetry';
+
+export class CopilotContextTelemetry {
+    private static readonly correlationIdKey = 'correlationId';
+    private static readonly copilotEventName = 'copilotContextProvider';
+    private readonly metrics: Record<string, number> = {};
+    private readonly properties: Record<string, string> = {};
+    private readonly id: string;
+    constructor(correlationId?: string) {

```

Comment:
There should be a blank newline before `constructor`.

Reply:
I'll add it manually Should this be enforced by whatever the formatter is? I think I have enabled ESLint and formatting the document should be the "correct" one.

Reply:
in principle formatting should be good now, let me know if I need to apply any fixes, thanks.